src/components/FooterNavList/FooterNavList.tsx

```diff
@@ -1,9 +1,36 @@
+import { Link } from 'react-router-dom';
+import styles from './footerNavList.module.css';
+
+interface NavItem {
+  label: string;
+  path?: string;
+}
+
 interface FooterNavListProps {
-  items: string[];
+  items: NavItem[];
 }
 
-const FooterNavList = (_: FooterNavListProps) => {
-  return <></>;
+const FooterNavList = ({ items }: FooterNavListProps) => {
+  const [firstItem, ...restItems] = items;
```

Comment:
This is wrong, just pass label separately

Reply:
Fixed